scripts/run_tests.sh

```diff
@@ -6,6 +6,8 @@ docker compose down
 # Start required docker instances
 docker compose up -d db db-migrations rabbitmq
 
+# Ensure services are up
+sleep 10
```

Comment:
Shouldn't docker compose healthcheck is not working?

Reply:
The health-check block the next container but is not blocking the end of the command.